modules/virt-creating-vm-instancetype.adoc

```diff
@@ -58,23 +62,58 @@ Logos of operating systems that are not available in the cluster are shown at th
+Immediately after you install the environment or when the environment is disconnected, the list of volumes to boot from is empty. In that case, three operating system logos are displayed: Windows, {op-system-base}, and Linux. You can add a new volume that meets your requirements by clicking the *Add volume* button.
+endif::[]
+
+ifndef::dynamic-key,virt-create-vms[]
+. Select a suitable bootable volume from the list. If the list is truncated, click the *Show all* button to display the entire list.
++
+[NOTE]
+====
+The bootable volume table lists only those volumes in the `openshift-virtualization-os-images` namespace that have the `instancetype.kubevirt.io/default-preference` label.
+====
+
+** Optional: Click the star icon to designate a bootable volume as a favorite. Starred bootable volumes appear first in the volume list.
+endif::[]
 
 ifdef::virt-create-vms[]
 . Click an instance type tile and select the resource size appropriate for your workload.
 endif::[]
 ifdef::dynamic-key[]
 . Click the *Red Hat Enterprise Linux 9 VM* tile.
 endif::[]
-. If you have not already added a public SSH key to your project, click the edit icon beside *Authorized SSH key* in the *VirtualMachine details* section.
-. Select one of the following options:
+ifdef::virt-create-vms[]
+. Optional: Choose the virtual machine details, including the VM's name, that apply to the volume you are booting from:
+endif::[]
+ifdef::virt-create-vms[]
+** For a Linux-based volume, follow these steps to configure SSH:
+endif::[]
 
+.. If you have not already added a public SSH key to your project, click the edit icon beside *Authorized SSH key* in the *VirtualMachine details* section.
+. Select one of the following options:
++
+--
 * *Use existing*: Select a secret from the secrets list.
-* *Add new*:
-.. Browse to the public SSH key file or paste the file in the key field.
-.. Enter the secret name.
-.. Optional: Select *Automatically apply this key to any new VirtualMachine you create in this project*.
+* *Add new*: Follow these steps:
+... Browse to the public SSH key file or paste the file in the key field.
+... Enter the secret name.
+... Optional: Select *Automatically apply this key to any new VirtualMachine you create in this project*.
+--
 .. Click *Save*.
+
+ifdef::virt-create-vms[]
+** For a Windows volume, follow either of these set of steps to configure sysprep options:
+*** If you have not already added sysprep options for the Windows volume, follow these steps:
+... Click the edit icon beside *Sysprep* in the *VirtualMachine details* section.
+... Add the *Autoattend.xml* answer file.
+... Add the *Unattend.xml* answer file.
+... Click *Save*.
+*** If you want to use existing sysprep options for the Windows volume, follow these steps:
+... Click *Attach existing sysprep*.
+... Enter the name of the existing sysprep *Unattend.xml* answer file.
+... Click *Save*.
+endif::[]
```

Comment:
Since the Linux-based path was an ordered/numbered step, I think these should flipped to ordered as well to match:
```suggestion
ifdef::virt-create-vms[]
. For a Windows volume, follow either of these set of steps to configure sysprep options:
** If you have not already added sysprep options for the Windows volume, follow these steps:
... Click the edit icon beside *Sysprep* in the *VirtualMachine details* section.
... Add the *Autoattend.xml* answer file.
... Add the *Unattend.xml* answer file.
... Click *Save*.
** If you want to use existing sysprep options for the Windows volume, follow these steps:
... Click *Attach existing sysprep*.
... Enter the name of the existing sysprep *Unattend.xml* answer file.
... Click *Save*.
endif::[]
```